### PR TITLE
refactor(components): rename titles hierarchy terminology

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

**Review by yinonov — COMMENTED** [review]
is this still WIP? the PR title somewhat differs from the change file title

**Review by yinonov — COMMENTED** [review]
@rachelbt card and note are the only affected components?

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Feel free to check [issues page](https://github.com/Vonage/vivid-3/issues). You
 
 ### Getting started
 
-#### Generator a new component 
+#### Generator a new component
```
Comment (YonatanKra) [inline README.md:56]:
```suggestion
#### Generate a new component
```

**Review by YonatanKra — CHANGES_REQUESTED** [review]
@rinaok - it seems like you've deleted the `contributing` and `ui-tests` docs in this PR. Have you moved them someplace else?

**Comment by rinaok** [comment]
> @rinaok - it seems like you've deleted the `contributing` and `ui-tests` docs in this PR. Have you moved them someplace else?

@YonatanKra Deleted by mistake

**Comment by rachelbt** [comment]
> > @rinaok - it seems like you've deleted the `contributing` and `ui-tests` docs in this PR. Have you moved them someplace else?
> 
> @YonatanKra Deleted by mistake

I added them back :)

**Review by rinaok — APPROVED** [review]

**Pull request merged**